apps/samples/Ensemble8/src/app/java/ensemble/samplepage/Description.java

```diff
@@ -31,26 +31,28 @@
  */
 package ensemble.samplepage;
 
-import ensemble.EnsembleApp;
-import ensemble.PlatformFeatures;
-import ensemble.SampleInfo;
-import ensemble.SampleInfo.URL;
-import ensemble.generated.Samples;
+import static ensemble.samplepage.SamplePage.INDENT;
+import static ensemble.samplepage.SamplePageContent.title;
```

Comment:
I see that the static imports (for ensemble.xxx) were moved up to the top. Is that deliberate?

Reply:
it's how the formatter is currently configured.
do we have a preferred order?